components/eventing-controller/controllers/backend/reconciler.go

```diff
@@ -622,7 +636,16 @@ func getSecretStringData(clientID, clientSecret, tokenEndpoint, grantType, publi
 }
 
 func (r *Reconciler) CreateOrUpdatePublisherProxy(ctx context.Context, backend eventingv1alpha1.BackendType) (*appsv1.Deployment, error) {
+	return r.CreateOrUpdatePublisherProxyDeployment(ctx, backend, true)
+}
+
+func (r *Reconciler) CreateOrUpdatePublisherProxyDeployment(
```

Comment:
can we make `CreateOrUpdatePublisherProxyDeployment` private?

Reply:
this is a new function used by Eventing Manager, and therefore it must be public. The reason I created a new function is that the old one `CreateOrUpdatePublisherProxy` sets EC deployment as owner, but we want Eventing CR as owner in case of Eventing Manager.

Reply:
Thank you!